stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.14.0
+version: 1.14.1
```

Comment:
As we are adding a feature please bump the minor version.
`1.15.0`

Reply:
bumped to `1.15.0`